local_planner/src/nodes/local_planner.cpp

```diff
@@ -65,7 +63,7 @@ void LocalPlanner::dynamicReconfigureSetParams(
   use_back_off_ = config.use_back_off_;
   use_VFH_star_ = config.use_VFH_star_;
   adapt_cost_params_ = config.adapt_cost_params_;
-  send_obstacles_fcu_ = config.send_obstacles_fcu_;
+  send_obstacles_fcu_ = model_params_.param_mpc_col_prev_d > 0.f;
```

Comment:
that's a bit wordy now, no? Is this the convention in the Firmware? Personally, I think the `model_params_` encodes the fact that they're params, so the members themselves don't really need to repeat that

Reply:
I really like the idea of keeping the same naming convention of the Firmware especially since it's going to be strictly enforced (https://github.com/PX4/Firmware/pull/11686).
What if we change `model_params_` to `px4`?

Reply:
Sure, even better! `model_params` isn't very clear to me anyway